config/example/config.txt

```diff
@@ -17,6 +17,13 @@ JOBS_HAVE_MINIMAL_ACCESS
 ## you have noone older than 0 days, since noone has been logged yet. Only turn this on once you have had the database up for 30 days.
 #USE_AGE_RESTRICTION_FOR_JOBS
 
+##Unhash this to track player playtime in the database. Requires database to be enabled.
+#USE_EXP_TRACKING
+##Unhash this to enable playtime requirements for jobs that have them defined.
+#USE_EXP_RESTRICTIONS
+##Unhash this to allow admins to bypass job playtime requirements.
+#USE_EXP_RESTRICTIONS_ADMIN_BYPASS
```

Comment:
This should either be
A) Uncommented by default
B) Changed to be an option of "DISABLE_EXP_RESTRICTION_ADMIN_BYPASS", and make the bypass on by default in the configuration controller.